lib/faraday_middleware/response/parse_json.rb

```diff
@@ -7,8 +7,18 @@ class ParseJson < ResponseMiddleware
       require 'json' unless defined?(::JSON)
     end
 
-    define_parser do |body|
-      ::JSON.parse body unless body.strip.empty?
+    define_parser do |body, options|
+      next if body.strip.empty?
+
+      ::JSON.parse(
+        body,
+        :max_nesting => options[:max_nesting],
+        :allow_nan => options[:allow_nan],
+        :symbolize_names => options[:symbolize_names],
+        :create_additions => options[:create_additions],
+        :object_class => options[:object_class],
+        :array_class => options[:array_class]
+      )
```

Comment:
This doesn't look DRY, it would be better to simply pass `options` as it is to `JSON.parse`.